routes/ci.js

```diff
@@ -191,11 +194,52 @@ router.post('/webhook', function(req, res, next) {
           status: true
         });
         break;
+      case "pull_request":
+        if (req.body.action === "reopened" || req.body.action ==="opened") {
+          var sha = req.body.pull_request.head.sha;
+          github.createStatus(sha, repositoryName, "pending", "Yaydoc is checking your build", process.env.PERSONAL_ACCESS_TOKEN, function (error, body) {
+            if (!error) {
+              var user = req.body.pull_request.head.label.split(":")[0];
+              var targetBranch = req.body.pull_request.head.label.split(":")[1];
+              var gitURL = `https://github.com/${user}/${req.body.repository.name}.git`;
+              console.log(gitURL);
+              console.log(targetBranch);
+              var data = {
+                email: "rbalajis25@gmail.com",
```

Comment:
Define a non-existent email, unless you want to receive the email message for every PR :wink: